ocamlbuild.install

```diff
@@ -0,0 +1,4 @@
+bin: [
+  "?ocamlbuild.native" {"ocamlbuild"}
+  "ocamlbuild.byte" {"ocamlbuild.byte"}
+]
```

Comment:
Agreed, that is how I understood it as well. Perhaps it's worth making an opam issue as well?